addons/mail/tests/test_res_partner.py

```diff
@@ -182,3 +182,86 @@ def test_res_partner_merge_wizards(self):
         all_msg = p2_msg_ids_init + p1_msg_ids_init + p1_msg1
         self.assertEqual(len(p2.message_ids), len(all_msg) + 1, 'Should have original messages + a log')
         self.assertTrue(all(msg in p2.message_ids for msg in all_msg))
+
+    def test_res_partner_get_starred_count(self):
+        # ignored message has (according to is_empty):
+        # - empty body OR
+        # - no subtype description OR
+        # - no tracking_value_id
+        Users = self.env['res.users']
+        new_user = Users.create({
+            'email': 'francoisemb@test.be',
+            'login': 'benjam',
+            'name': 'Benjam Prejent',
+        })
+        new_partner = new_user.partner_id
+        new_msg = new_user.message_ids
+        subtype_test = self.env['mail.message.subtype'].create({
+            'name': 'TestSubtype',
+            'description': 'PHP is the best programming language after HTML'
```

Comment:
I don't want to be responsible of approving such heretic message.

And even if I liked the message, I think it would be better to take an actual realistic example of subtype description to better understand the business idea behind the test.